eng/jobs/windows-build.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     pool:
       # Use a hosted pool when possible.
       ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        name: NetCorePublic-Pool
-        queue: buildpool.windows.10.amd64.vs2019.open
+        name: NetCore1ESPool-Svc-Public
```

Comment:
This is targeting main but you're using servicing pools, you should use R&D pools

Reply:
Ah, thank you ulises, that's my bad. Will fix.